passenger_service/src/main/java/com/intership/passenger_service/controller/CommandPassengerController.java

```diff
@@ -0,0 +1,33 @@
+package com.intership.passenger_service.controller;
+
+import com.intership.passenger_service.dto.ProfileDto;
+import com.intership.passenger_service.service.CommandPassengerProfileService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/passenger")
+@RequiredArgsConstructor
+public class CommandPassengerController {
+    private final CommandPassengerProfileService profileService;
+
+    @PostMapping("/save-passenger")
```

Comment:
POST запрос уже означает, что ты сохраняешь сущность
```suggestion
    @PostMapping
```